docs/plone-deployment/new-project.md

```diff
@@ -142,3 +141,18 @@ When the process completes successfully, it will exit with a message similar to
 ```console
 ✨  Done in 98.97s.
 ```
+
+Remember, having the repository on GitHub is currently a requirement to build the Docker images automatically; this might be a good moment to do so in case you had not yet.
+
+The same could be done with other options like GitLab, but we do not provide a working example for it at the moment.
+
+````{note}
+Due to an output difference when translations are built by the `@plone/generator-volto`, at the moment, it is necessary to run on the root directory:
+
+```{code-block} shell
```

Comment:
`code-block` is necessary only when you want to use reStructuredText features, such as line numbering and emphasize lines.

````suggestion
```shell
````